### ISSUE TITLE
https://docs.clamav.net/ has visual bug making it difficult to read

### ISSUE THREAD

**Issue by benmordecai** [Cisco-Talos/clamav#1080]
Describe the bug
----------------

Using Brave v1.60.110 and LibreWolf 119.0-5, docs.clamav.net has a permanently present navigator bar that obscures the text making it illegible.

![image](https://github.com/Cisco-Talos/clamav/assets/2068192/75cbc3c6-3c40-43b9-91ac-d062f6e75855)

**Comment by micahsnyder** [comment]
It is probably a bug [in `mdbook`](https://github.com/rust-lang/mdBook), though it may be something that we've customized with css ...

Can you take a look for other mdbook-based sites and check if they have the same issue?

**Comment by micahsnyder** [comment]
Perhaps because I increased the max width to make it show the whole table: https://github.com/Cisco-Talos/clamav-documentation/commit/f9cb3616238738fbfb5dda038e4abd39ae02daa1#diff-91dddace8f0ad8c43f1f4d6fa3ddd645b85ba6fa03f51426d0330f133e3f1928

I do see that the way the menu on the left is structured differently for docs.clamav.net:
![image](https://github.com/Cisco-Talos/clamav/assets/30635813/a7c708bb-0be4-4576-9957-0409a74f9e02)

versus:
![image](https://github.com/Cisco-Talos/clamav/assets/30635813/d7fe9f74-210c-45bd-ac19-6d1efe7e4a79)

And for docs.clamav.net I'm also unable to minimize the menu when clicking the pancake-stack in the top left. 
Hmm I wonder what's going on...

**Comment by benmordecai** [comment]
Let me know if there is a way I can help.

**Comment by benmordecai** [comment]
Solved

**Comment by micahsnyder** [comment]
> Solved

Uh.... what happened?

**Comment by benmordecai** [comment]
Magically worked today with no changes.

**Comment by benmordecai** [comment]
As a test, I dropped my scaling to 100% and the visual glitch is gone. Perhaps the layout takes into account display resolution but not scaling, and it assumes 100% scaling?

**Comment by micahsnyder** [comment]
My teammate is seeing the same issue at 100%.  Very strange.  Will see if we can fix it.

**Comment by micahsnyder** [comment]
It seems to be this issue https://github.com/rust-lang/mdBook/issues/2217

I can't reproduce the issue locally, however. When I do `mdbook build` and then either `mdbook serve` or just open the `index.html` file it all looks normal.  Not sure what's going on.  I'll try to force it to rebuild.

**Comment by micahsnyder** [comment]
Oh nevermind I didn't have the latest `mdbook` version on my local machine.  When I replaced package `mdbook v0.4.32` with `mdbook v0.4.35` and then did the build I found that the sidebar is no longer collapsible and the sidebar sits overtop of the text when the browser window isn't wide enough. 

Guessing by the comments in the closed mdbook issue, It's possible that some update is needed to the theme to fix it.  But I'm going to set our github actions to use an older version of mdbook for now and raise the issue with the mdbook devs.
